.github/dependabot.yml

```diff
@@ -90,6 +90,16 @@ updates:
     interval: daily
     time: "06:00"
 
+- package-ecosystem: "gomod"
+  directory: "/contrib/golang/filters/http/test/test_data/buffer_flush"
```

Comment:
@spacewander there is a current conversation about consolidating/rationalizing these deps to one

currently the go test deps all contain the same deps and whenever they change (ie protobuf) we get an avalanche of dependency updates (and ci)

any chance we could address this first? cc @doujiang24

Reply:
Yep, just created a PR: https://github.com/envoyproxy/envoy/pull/37962, PTAL cc @phlax

Reply:
#37962 has landed - so iiuc this bit should no longer be necessary